README.md

```diff
@@ -27,7 +27,7 @@ The recommended pipeline to run Pino GELF as a transform for Pino logs is as fol
 node your-app.js | pino-gelf log
 ```
 
-The host, port and maximum chunk size of your Graylog server can be specified using options. Options can also be used to include standard fields logged by [express-pino-middleware](https://github.com/pinojs/express-pino-logger). Custom fields included with Pino requests based on your own configuration can also be specified, although at present only strings are handled. Finally, you can choose to enable verbose mode which outputs all GELF messages locally, this can be useful for initial configuration.
+The host, port and maximum chunk size of your Graylog server can be specified using options. Fields logged by [express-pino-middleware](https://github.com/pinojs/express-pino-logger) or any other custom fields included with Pino requests will be converted into strings (if they aren't already) and their keys will be prefixed with an underscore, according to the GELF spec. Also, you can choose to enable verbose mode which outputs all GELF messages locally, this can be useful for initial configuration. Finally, you can enable passthrough mode which will output the original input back to stdout. That enables you to make `pino-gelf` part of a chain of commands and use the original input for further processing (for example `pino-pretty`).
```

Comment:
this has nothing to do with gelf correct? this is just an additional convenience feature? does anything else happen when passthough mode is enabled - e.g. is it still sending data somewhere?

Reply:
I think maybe this should go in its own heading

Reply:
@davidmarkclements Yes, putting passthrough in its own heading is a good idea, I'll take care of it.

Reply:
@davidmarkclements I've decided not to put it into its own heading but into its own paragraph instead. It's still under "usage" where I believe it belongs. And I had already bumped the version up to 2.0.0.

Reply:
awesome